backend/server.py

```
@@ -7385,6 +7199,28 @@ async def scheduler_force_cleanup(user: dict = Depends(get_current_user)):
         raise HTTPException(status_code=500, detail=str(exc))
 
 
+@app.post("/api/scheduler/self-heal")
+async def scheduler_self_heal(user: dict = Depends(get_current_user)):
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)